### ISSUE TITLE
Staging Review finding: Headings not nested correctly

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#101662]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** MHV Medications
**Product:** Prescription grouping
**Feature:** Linking similar meds

## Findings details
**VA.gov Experience Standard - issue:** [User can't determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When viewing a medication details page, the heading structure doesn't seem to quite align with the actual structure of the information.

The current heading structure is something like:
- H1: [Name of medication]
   - H2: Most recent prescription
      - H3: Prescription number
      - ... (more stuff about this prescription)
      - H3: Prescribed by 
      - H3: Refill history
         - H4: Refill, Filled on [date] (button)
         - H4: Image
         - H4: Medication description
      - H3: Previous prescriptions
         - H4: Prescription number: [number]

I think there are two issues:
- As I read the content of the page, there are two big sections of information about the medication --- the most recent prescription and the previous prescriptions. Currently, previous prescriptions are nested under the most recent prescription.
- Each refill/original fill accordion's toggle button is an H4, but so are the headings inside the accordion.

## Recommended action

I think what would make more sense would be:
- H1: [Name of medication]
   - H2: Most recent prescription
      - H3: Prescription number
      - ... (more stuff about this prescription)
      - H3: Prescribed by 
      - H3: Refill history
         - H4: Refill, Filled on [date] (button)
            - H5: Image
            - H5: Medication description
   - H2: Previous prescriptions
      - H3: Prescription number: [number]

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by IdeaTrail** [comment]
This issue is being addressed by Jira item[MHV-66237](https://jira.devops.va.gov/browse/MHV-66237)

**Comment by TannerBird98** [comment]
Ticket is in ready for validation and should be ready to be closed in the next few days

**Comment by IdeaTrail** [comment]
https://jira.devops.va.gov/browse/MHV-66237 has been completed.
